projects/gis/src/app/main/main.component.html

```diff
@@ -33,7 +33,7 @@
     <geoengine-layer-list
         class="mat-elevation-z4"
         [height]="middleContainerHeight$ | async | numberSanitizer"
-        [addAFirstLayerSidenavConfig]="addDataConfig | async | valueDefault: {component: AddDataComponent}"
+        [addAFirstLayerSidenavConfig]="addDataConfig | async | valueDefault: undefined"
```

Comment:
This has to be done in the gfbio app.component.html as well

Reply:
[02ecf14](https://github.com/geo-engine/geoengine-ui/pull/496/commits/02ecf1496c92c6f7739bbf6a7621bdb6dbe14078)